app/styleguide/wskComponentHandler.js

```diff
@@ -75,6 +75,11 @@ var componentHandler = (function() {
       var registeredClass = findRegisteredClass_(jsClass);
       if (registeredClass) {
         createdComponents_.push(new registeredClass.classConstructor(element));
+        // Call any callbacks the user has registered with this component type.
+        var length = registeredClass.callbacks.length;
+        for (var n = 0; n < length; n++) {
+          registeredClass.callbacks[n](element);
+        }
```

Comment:
`.forEach` is better in this case because it's shorter, more readable and explicit about the intent (I'm surprised you don't think so), meaningless perf gain with for-loop, less chance of an off by one error.

> In addition I am following the Google Style guide which recommends using normal for loops with array based data.

Source? Does it explicitly recommend using normal for-loop over `.forEach`?

-

I'm not looking to waste more time bikeshedding this. Do whatever you like. Was just an educated suggestion. I prefer succinctness over premature optimization any day.